### ISSUE TITLE
Retroarch is slow only when launched from big picture mode. It works fine and fast from steamos desktop.

### ISSUE THREAD

**Issue by 128e** [ValveSoftware/SteamOS#466]
Retroarch and its emulators run horribly slow when launched via the steamOS big picture mode, but when launched from the Desktop mode they run full speed. this is true of the retroarch menu and of the emulation itself.

This may be something to do with steam overlay? but i'm not sure, is there any way to disable steam overlay to test this?

I have checked the framerate to confirm this issue.

My system: i7 960 8GB RAM AMD 7790 2GB GPU

**Comment by aboutafter** [comment]
Yes, the performance when using the Big Picture Mode overlay is atrocious. It looks like there is some kind of incompatibility between the BPM overlay and the RetroArch one. 

It could be a good idea to add this issue to the RetroArch issues tracker too.

**Comment by ProfessorKaos64** [comment]
This isn't something supported by SteamOS, and not a bug. Do you think they are going to phone up  the libretro devs to fix two things which are not a focus of SteamOS?

**Comment by 128e** [comment]
I'm not certain if this problem lies with steamOS or with the retroarch guys. It seems like an incompatibility with steamOS as it doesn't happen in anything else.

however i would say supporting the ability to play all the great classic couch games on steamOS should probably be a priority.

**Comment by nerdbeard** [comment]
@ProfessorKaos64 -- How can you say this is a bug in RetroArch and not in BPM without any evidence whatsoever?  If RetroArch can't run due to a bug in BPM, it's only a matter of time until more titles (perhaps ones in the Steam store right now) are discovered.  There is an obvious interest to Valve in fixing *all* bugs in Steam.

**Comment by ProfessorKaos64** [comment]
> If RetroArch can't run due to a bug in BPM

Works fine for me on my system (Intel i5 / GTX 770) and I can plan Dolphin games just fine on my GT 640 low profile test machine paired with a Core 2 Quad Q9550 (old cpu). I know this is not everyone.

> here is an obvious interest to Valve in fixing all bugs in Steam.

Fair enough. I think it's known though that the overlay impacts performance. I'd hazard a guess that Valve devs are aware of FPS drops. How is performance on Steam For Linux, using BPM, on other* distros?

**Comment by aaronbond** [comment]
I'll submit an issue to the retroarch guys as well, and see what they think. It's either the fact that it's an overlay running within an overlay or something else.
